docs/shared/recipes/plugins/create-package.md

```diff
@@ -0,0 +1,75 @@
+# Creating a create package
+
+When a package named create-{x} is published to npm, it can be invoked via any of the following:
+
+- npx create-{x}
+- yarn create {x}
+- npm init {x}
+- pnpm init {x}
+
+These packages are used to set up a new project in some form. As an example, consider `create-nx-plugin`, `create-nx-workspace`, or perhaps `create-react-app`.
+
+There are a few paths to create a package that will work with `create-nx-workspace`'s public API to setup a new workspace that uses your Nx plugin:
+
+- When creating a new nx plugin via `create-nx-plugin` to setup a workspace, provide a name when prompted for your create package
+- Run `nx g create-package create-my-plugin --project my-plugin` to add a new cli tool called `create-my-plugin` that is setup for `my-plugin`.
+
+In either case, you'll have 2 packages that are relevant in your workspace:
+
+- The plugin package
+- The create package
+
+You should also have a generator called `preset` that was generated within the plugin package. This preset generator will be invoked when running either `npx create-nx-workspace --preset my-plugin` or `npx create-my-plugin`. For more information about customizing your preset, see: [Creating a Preset](/plugins/recipes/create-preset).
+
+Let's take a look at the code that was scaffolded out for your create-my-plugin package:
+
+```typescript {% fileName="packages/create-my-plugin/bin/index.ts" %}
+#!/usr/bin/env node
+
+import { createWorkspace } from 'create-nx-workspace';
+
+async function main() {
+  const name = process.argv[2]; // TODO: use libraries like yargs or enquirer to set your workspace name
+  if (!name) {
+    throw new Error('Please provide a name for the workspace');
+  }
+
+  console.log(`Creating the workspace: ${name}`);
+
+  // This assumes "my-plugin" and "create-my-plugin" are at the same version
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const presetVersion = require('../package.json').version;
+
+  // TODO: update below to customize the workspace
+  const { directory } = await createWorkspace(`my-plugin@${presetVersion}`, {
+    name,
+    nxCloud: false,
+    packageManager: 'npm',
+  });
+
+  console.log(`Successfully created the workspace: ${directory}.`);
+}
+
+main();
+```
+
+By utilizing your custom preset to handle the generation, we can provide equal functionality between `create-nx-workspace` and your new create-package.
+
+This is a plain node script at this point, and you can use any dependencies you wish to handle things like prompting or argument parsing. Keeping dependencies small and splitting out the command line tool from the Nx plugin is recommended, and will help keep your CLI feel fast and snappy.
```

Comment:
"Splitting out the command line tool from the Nx plugin" - you'll need to give more details about what you mean or not mention this at all.  I don't know exactly what you're suggesting